Mathlib/MeasureTheory/Integral/FundThmCalculus.lean

```diff
@@ -1243,6 +1243,28 @@ theorem integral_deriv_eq_sub' (f) (hderiv : deriv f = f')
   exact hcont.intervalIntegrable
 #align interval_integral.integral_deriv_eq_sub' intervalIntegral.integral_deriv_eq_sub'
 
+/-- A variant of the Fundamental theorem of calculus-2 involving integrating over the
+unit interval. -/
+lemma integral_unitInterval_eq_sub {C E : Type*} [NontriviallyNormedField C]
+    [NormedAlgebra ℝ C] [NormedAddCommGroup E] [NormedSpace ℝ E] [NormedSpace C E]
```

Comment:
Does it apply to any `C` other than real and complex numbers? If no, then use `[IsROrC C]` instead of current assumptions.

Reply:
Probably not. I have changed it to use `IsROrC`.